src/framebuffers.h

```diff
@@ -5,10 +5,10 @@
 #include "config.h"
 
 extern u8 D_802BA290[22768];
-extern u8 D_802BFB80[77824];
+//extern u8 D_802BFB80[77824];
```

Comment:
Why is this commented out?

Reply:
Long story short it was to try and rectify a problem with including both variables.h and framebuffers.h in the same file. I've found a different way to address that problem in this PR, but I'm going to try and address the problem head on in a later PR.